jenkins_job_wrecker/modules/triggers.py

```diff
@@ -38,18 +40,211 @@ def reversebuildtrigger(top, parent):
+                        branches = []
+                        for branch_type in attribute:
+                            if branch_type.tag == "com.sonyericsson.hudson.plugins.gerrit.trigger.hudsontrigger.data.Branch":
+                                branch = {}
+                                for branch_attribute in attribute[0]:
+                                    if branch_attribute.tag == "compareType":
+                                        branch["branch-compare-type"] = branch_attribute.text
+                                    elif branch_attribute.tag == "pattern":
+                                        branch["branch-pattern"] = branch_attribute.text
+                                    else:
+                                        raise NotImplementedError("Not implemented branch attribute: ",
+                                                                  branch_attribute.tag)
+                                branches.append(branch)
+                            else:
+                                raise NotImplementedError("Not implemented branch type: ", branch_type.tag)
+                        project["branches"] = branches
+
+                    elif attribute.tag == "disableStrictForbiddenFileVerification":
+                        project["disable-strict-forbidden-file-verification"] = get_bool(attribute.text)
+                    elif attribute.tag == "filePaths":
+                        file_paths = process_file_paths(attribute)
+                        project["file-paths"] = file_paths
+                    elif attribute.tag == "forbiddenFilePaths":
+                        forbidden_file_paths = process_file_paths(attribute)
+                        project["forbidden-file-paths"] = forbidden_file_paths
+                    elif attribute.tag == "topics":
+                        topics = process_file_paths(attribute)
+                        project["topics"] = topics
+                    else:
+                        raise NotImplementedError("Not implemented attribute: ", attribute.tag)
+
+                projects.append(project)
+            gerrit_trigger["projects"] = projects
+        elif child.tag == "dynamicGerritProjects":
+            pass
+        elif child.tag == "spec":
+            pass
+        elif child.tag == "skipVote":
+            skip_vote = {}
+            for attribute in child:
+                if attribute.text == "onSuccessful":
+                    skip_vote["successful"] = get_bool(attribute.text)
+                if attribute.text == "onFailed":
+                    skip_vote["failed"] = get_bool(attribute.text)
+                if attribute.text == "onUnstable":
+                    skip_vote["unstable"] = get_bool(attribute.text)
+                if attribute.text == "onNotBuilt":
+                    skip_vote["notbuilt"] = get_bool(attribute.text)
+            gerrit_trigger["skip_vote"] = skip_vote
+        elif child.tag == "gerritBuildStartedVerifiedValue":
```

Comment:
I'd reduce the repetitive work here like in my commit: https://github.com/romanek-adam/jenkins-job-wrecker/commit/2b527399d6866439b0d4e0acc3d247b4b05f4875